src/main/kotlin/blackjack/application/BlackJackParticipants.kt

```diff
@@ -0,0 +1,65 @@
+package blackjack.application
+
+import blackjack.application.dto.BlackJackStatuses
+import blackjack.application.dto.BlackJackWinningResults
+import blackjack.application.dto.BlackjackScores
+import blackjack.domain.card.CardDeck
+import blackjack.domain.participant.dealer.Dealer
+import blackjack.domain.participant.dealer.DealerScoreStrategy
+import blackjack.domain.participant.dealer.blackJackScore
+import blackjack.domain.participant.dealer.status
+import blackjack.domain.participant.dealer.winningResult
+import blackjack.domain.participant.player.Player
+import blackjack.domain.participant.player.PlayerScoreStrategy
+import blackjack.domain.participant.player.Players
+import blackjack.domain.participant.player.blackJackScores
+import blackjack.domain.participant.player.statuses
+import blackjack.domain.participant.player.winningResults
+import blackjack.domain.participant.vo.Name
+
+class BlackJackParticipants(
+    private val dealer: Dealer,
+    private val players: Players
+) {
+
+    val statuses: BlackJackStatuses
+        get() = BlackJackStatuses(listOf(dealer.status()) + players.statuses())
```

Comment:
순서를 바꾸면 단일 객체를 listOf로 감싸지 않을 수 있습니다 :)

```suggestion
    val statuses: BlackJackStatuses
        get() = BlackJackStatuses(players.statuses() + dealer.status())
```

Reply:
오호 list.add() 로 자동변환 해주나 보네요..!